airflow/providers/oracle/hooks/oracle.py

```diff
@@ -90,8 +91,19 @@ def get_conn(self) -> 'OracleHook':
                 dsn = conn.host
                 if conn.port is not None:
                     dsn += ":" + str(conn.port)
-                if service_name or conn.schema:
-                    dsn += "/" + (service_name or conn.schema)
+                if service_name:
+                    dsn += "/" + service_name
+                elif conn.schema:
+                    self.log.warning(
+                        """You are using the parameter conn.schema to pass the Service Name
+                        of your Oracle Database. In this version of the Hook, conn.schema
+                        is additionally used to set the 'current_schema' attribute of the
+                        Oracle DB Session.
+                        Please use the correct parameter conn.extra.service_name instead to
+                        prevent future issues.
+                        More Info: https://github.com/apache/airflow/pull/19084"""
+                    )
```

Comment:
Can we make this a `DeprecationWarning`? There are multiple instances you can search as examples.

Reply:
Sure, will look at it!
I also thought of shortening the text a little. Maybe to this:
```
Using conn.schema to pass the Oracle Service Name is deprecated.
Please use conn.extra.service_name instead.
More Info: https://github.com/apache/airflow/pull/19084
```
What do you think about this?

Reply:
Probably don’t need the More Info part; the first two sentences are good enough for me.

Reply:
Hi @uranusjr, sorry it took so long.
I changed the warning to `DeprecationWarning` and simplified the message.